tests/src/test/scala/org/apache/openwhisk/core/loadBalancer/test/ShardingContainerPoolBalancerTests.scala

```diff
@@ -288,14 +300,19 @@ class ShardingContainerPoolBalancerTests
       ShardingContainerPoolBalancer
         .schedule(1, fqn, invokers, invokerSlots, 1, index = 0, step = 1)
         .get
+        ._1
         .toInt
     }
 
     result shouldBe expectedResult
 
     // more schedules will result in randomized invokers, but the unhealthy and offline invokers should not be part
     val bruteResult = (0 to 100).map { _ =>
-      ShardingContainerPoolBalancer.schedule(1, fqn, invokers, invokerSlots, 1, index = 0, step = 1).get.toInt
+      ShardingContainerPoolBalancer
+        .schedule(1, fqn, invokers, invokerSlots, 1, index = 0, step = 1)
+        .get
+        ._1
+        .toInt
```

Comment:
This test should also verify that `_2` is actually `true`

Reply:
Thanks for the comments @markusthoemmes. Will work on them 👍